sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/schema.yaml

```diff
@@ -0,0 +1,157 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+  description: Date of the visit
+- mode: NULLABLE
+  name: visit_identifier
+  type: STRING
+  description: Visit Identifier - Uniquely identifies a visit; concatenation of user_pseudo_id and ga_session_id
+- mode: NULLABLE
+  name: full_visitor_id
+  type: STRING
+  description: Full Visitor ID - Uniquely identifies a visitor - this is the same as GA4 user_pseudo_id
+- mode: NULLABLE
+  name: visit_start_time
+  type: INT64
+  description: Visit Start Time - The event timestamp from the first event in the visit
+- mode: NULLABLE
+  name: page_path
+  type: STRING
+  description: Page Path
+- mode: NULLABLE
+  name: page_path_level1
+  type: STRING
+  description: Page Path Level 1
+- mode: NULLABLE
+  name: hit_type
+  type: STRING
+  description: Hit Type - PAGE if event_name is a page view, EVENT for all other event types
+- mode: NULLABLE
+  name: is_exit
+  type: BOOLEAN
+  description: Is Exit - The specific page view event that is considered the exit to the visit
+- mode: NULLABLE
+  name: is_entrance
+  type: BOOLEAN
+  description: Is Entrance - The specific page view event that is considered the entrance to the visit
+- mode: NULLABLE
+  name: hit_number
+  type: INT64
+  description: Hit Number - Densely ranked, since there can be multiple events at the same time for a user
+- mode: NULLABLE
+  name: hit_timestamp
+  type: INT64
+  description: Hit Timestamp - Same as the "Event Timestamp"
+- mode: NULLABLE
+  name: event_category
+  type: STRING
+  description: Event Category
+- mode: NULLABLE
+  name: event_name
+  type: STRING
+  description: Event Name
+- mode: NULLABLE
+  name: event_label
+  type: STRING
+  description: Event Label
+- mode: NULLABLE
+  name: event_action
+  type: STRING
+  description: Event Action
+- mode: NULLABLE
+  name: device_category
+  type: STRING
+  description: Device Category - The device category the visitor used to visit the site
+- mode: NULLABLE
+  name: operating_system
+  type: STRING
+  description: Operating System - The operating system the visitor used to visit the site
+- mode: NULLABLE
+  name: language
+  type: STRING
+  description: Language - The language the visiting device was using when it visited the site
+- mode: NULLABLE
+  name: browser
+  type: STRING
+  description: Browser - The browser the visiting device was using when it visited the site
+- mode: NULLABLE
+  name: browser_version
+  type: STRING
+  description: Browser Version - The version that the visiting device's browser was using when it visited the site
+- mode: NULLABLE
+  name: country
+  type: STRING
+  description: Country - The country from which events were reported, based on IP address
+- mode: NULLABLE
```

Comment:
The attribution fields source, medium, campaign, are prefixed by attribution in our analytics tables.
Mind adding the descriptions so they appear in the Data Catalog? See this [schema for reference](https://github.com/mozilla/bigquery-etl/blob/dd8d736c99bd9b4efd6b666e4fe424b86029bd20/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/schema.yaml#L76-L98).

Reply:
Sure, will do - thanks! I didn't know what those were, this is helpful